clearlinux/README-short.txt

```diff
@@ -0,0 +1 @@
+Official docker build of Clear Linux OS for Intel Architecture
```

Comment:
It's the official name we use (either OS or Project, in this case OS made more sense to me).